packages/synchro-charts/src/components/sc-widget-grid/sc-grid-tooltip.tsx

```diff
@@ -18,8 +21,17 @@ export class ScGridTooltip {
   @Prop() propertyPoint?: DataPoint;
   @Prop() alarmPoint?: DataPoint;
   @Prop() breachedThreshold?: Threshold;
+  @Prop() unit?: string;
+  @Prop() value?: number | string;
+
+  @Prop() messageOverrides: RecursivePartial<TooltipMessage>;
 
   private tooltip: Instance | undefined;
+  private messages: TooltipMessage;
+
+  componentWillLoad() {
+    this.messages = merge(DefaultTooltipMessages, this.messageOverrides);
```

Comment:
if someone switches language, will this work still?

Reply:
It works to all languages. 
If someone wants switch to another language or customize scripts. It has to be transferred through `messageOverrides`.